unlock-app/package.json

```diff
@@ -39,31 +49,40 @@
     "eslint-plugin-react": "^7.7.0",
     "eslint-plugin-standard": "^4.0.0",
     "expect-enzyme": "^1.3.0",
+    "jest": "^23.6.0",
     "jest-enzyme": "^7.0.0",
     "nock": "^10.0.1",
+    "nodemon": "^1.12.1",
+    "react-addons-test-utils": "15.6.2",
     "react-router-test-context": "^0.1.0",
+    "react-test-renderer": "16.4.2",
     "redux-mock-store": "^1.5.1",
-    "storybook-react-router": "^1.0.1"
+    "storybook-react-router": "^1.0.1",
+    "webpack": "^4.20.2"
   },
   "engines": {
     "node": "=8.11.4"
   },
   "engineStrict": true,
   "scripts": {
-    "start-app": "react-scripts start",
+    "dev": "npm run prestart && nodemon src/server.js",
+    "build": "next build src",
+    "start": "NODE_ENV=production node src/server.js",
     "start-ganache": "ganache-cli -b 3 -i 1984 --deterministic &",
     "deploy-lock": "cd ../smart-contracts && truffle deploy --reset",
-    "prestart": "npm run start-ganache && npm run deploy-lock && npm run serve-unlockjs",
```

Comment:
Could you tell me why you removed `npm run serve-unlockjs`? 
This serves the snippet of code that is being "embedded" by 3rd party sites. See https://staging.unlock-protocol.com/unlock.js